packages/mobile/src/screens/collection-screen/CollectionScreen.tsx

```diff
@@ -68,32 +68,12 @@ const useStyles = makeStyles(({ spacing }) => ({
  */
 export const CollectionScreen = () => {
   const { params } = useRoute<'Collection'>()
+  const { id, searchCollection, slug, collectionType } = params ?? {}
   const dispatch = useDispatch()
 
-  // params is incorrectly typed and can sometimes be undefined
-  const {
-    id: idParam,
-    searchCollection,
-    collectionName,
-    collectionType
-  } = params ?? {}
-
-  const id = useMemo(() => {
-    if (collectionName) {
```

Comment:
curious if you think we should keep this around for that edge case you mentioned earlier where playlists in local storage might not have permalinks yet?

Reply:
Yeah exactly what I was thinking as well, unless it's the case that the actions/sagas we are running take the permalink and extract handle id as a backup? Guessing that's not the case and I should send all three along in the fetch action?

Reply:
yeah that is not the case, so we'd have to send all 3 along in fetch 😬 yea i think we'd want to pass along in fetch? also realizing this is the screen used for albums too probably, and albums use the original routing and expect the name to contain id and name so we still need this for sure